modules/book.js

```diff
@@ -0,0 +1,39 @@
+const bookList = document.querySelector('.book-list');
+
+export default class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+
+  addBook() {
+    Book.books.push(this);
+    localStorage.setItem('bookInfo', JSON.stringify(Book.books));
+    Book.displayUI();
+  }
```

Comment:
- [ ] Wonderful work you have done on this project, it would be nice to convert this function to _**ES6 arrow function**_. This way, you would be fulfilling the project requirements. 😄